tdrs-backend/init.sql

```diff
@@ -0,0 +1,6 @@
+CREATE ROLE readaccess;
+GRANT CONNECT ON DATABASE tdrs_test TO readaccess;
```

Comment:
parametrize db name

Reply:
make sure it selects the correct database depending on the execution environment